test/selenium/print_test.js

```diff
@@ -0,0 +1,14 @@
+// 03 Print test using browserstack
+
+var webdriver = require('browserstack-webdriver');
+
+var runTest = function(cap, driver, target) {
+  // Click on "Drucken"
+  driver.findElement(webdriver.By.xpath("//a[@id='printHeading']")).click();
+  // Wait 1 sec for the menu to pull down
+  driver.sleep(1000);
```

Comment:
We should not use sleep in the tests. There should be a way to test for existence of an element (or class or attribute) that changes which indicates that the pull down is open.

Here, you can see if the element with id `printHeading` has the class `collapsed`. This should be possible with a findElement call.